Makefile

```
@@ -144,12 +144,15 @@ requirements.txt:	pyproject.toml uv.lock ## Generate requirements.txt file conta
 verify-packages-completeness:	requirements.txt ## Verify that requirements.txt file contains complete list of packages
 	pip download -d /tmp/ --use-pep517 --verbose -r requirements.txt
 
-get-rag: ## Download a copy of the RAG embedding model and vector database
-	podman create --replace --name tmp-rag-container $$(grep 'ARG LIGHTSPEED_RAG_CONTENT_IMAGE' Containerfile | awk 'BEGIN{FS="="}{print $$2}') true
-	rm -rf vector_db embeddings_model
-	podman cp tmp-rag-container:/rag/vector_db vector_db
-	podman cp tmp-rag-container:/rag/embeddings_model embeddings_model
-	podman rm tmp-rag-container
+get-embeddings: get-embeddings-byok get-embeddings-okp ## Download all embedding model binaries
```

checkmake (0.3.2)

[warning] Target "get-embeddings" should be declared PHONY.

(phonydeclared)